strawberry/utils/mixins.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+from strawberry.utils.str_converters import to_camel_case
+
+
+class GraphQLNameMixin:
+    python_name: str
+    graphql_name: Optional[str]
```

Comment:
Couldn't a StrawberryField not have a `python_name` if defined using a lambda and an explicit `name` arg?

Reply:
how would you define a field using a lambda?

Reply:
Oh right, that would be a resolver